conda/cli/main_info.py

```diff
@@ -206,7 +234,14 @@ def print_package_info(packages):
     )
 
 
-def get_info_dict(system=False):
+def get_info_dict(system: bool = False) -> dict[str, Any]:
+    """
+    Returns a dictionary of contextual information.
+
+    :param system:  Unused.
```

Comment:
Since it's unused we want to deprecate it:

```suggestion
@deprecated.argument("24.9", "25.3", "system")
def get_info_dict() -> dict[str, Any]:
    """
    Returns a dictionary of contextual information.

```